datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java

```diff
@@ -982,13 +1014,54 @@ public void onBindingDied(final @NonNull ComponentName name) {
         @Override
         public void handleMessage(final @NonNull Message msg) {
             Log.v(TAG, String.format("Service facade received message: %d", msg.what));
+            final Bundle parcel;
+            parcel = msg.getData();
+            parcel.setClassLoader(getClass().getClassLoader());
+
+            // The Broadcasts for the ShutdownFinishedHandler need to be independent from the number of registered
+            // CapturingListeners
+            switch (msg.what) {
+                case MessageCodes.SERVICE_STOPPED:
+                    // Due to the <code>DataCapturingBackgroundService#informCaller()</code> interface
```

Comment:
You do not need those <code>-Tags in normal comments. They are not formatted as HTML.